src/Microsoft.AspNetCore.Sockets.Http/Internal/Transports/WebSocketsTransport.Log.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.WebSockets;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Sockets.Internal.Transports
+{
+    public partial class WebSocketsTransport
+    {
+        private static class Log
+        {
+            private static readonly Action<ILogger, Exception> _socketOpened =
+                LoggerMessage.Define(LogLevel.Information, new EventId(1, "SocketOpened"), "Socket opened.");
```

Comment:
We `Info` trace a lot. Should this be demoted to `Debug`?

Reply:
I don't think Transports should emit Info level messages at all. Changing them to Debug. Fite me irl

Reply:
Sounds good to me.